src/Register.js

```diff
@@ -1,24 +1,52 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { View, Text, TouchableOpacity, StyleSheet } from 'react-native'
 import Btn from './Btn'
 import Field from './Field'
-import { darkGreen } from './Constants'
+import { darkGreen, red } from './Constants'
 
 const Register = (props) => {
+  const [username, setUsername] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [errorMsg, setErrorMsg] = useState("");
+
+  const register_api = async () => {
+
+    await fetch('http://127.0.0.1:8000/api/register', {
```

Comment:
This should be an environment variable, it will not work when pointing it to `localhost`

Reply:
That is exactly what I meant
See if [this](https://medium.com/swlh/how-to-properly-use-environment-variables-in-an-expo-react-native-app-7ab852590b30#:~:text=What%20is%20an%20Environment%20Variable,of%20your%20React%20Native%20App.) helps you